geometry_msgs/Pose[] target_waypoints
float64 force_threshold
---
bool success
string error_msg